features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumAttributes.kt

```diff
@@ -287,4 +287,21 @@ object RumAttributes {
     const val NETWORK_BYTES_READ: String = "network.bytes_read"
 
     // endregion
+
+    // region Internal attributes
+
+    /**
+     * Custom Flutter vital - First Build Complete. The amount of time between a route change (the start of a view)
+     * and when the first `build` method is complete. In nanoseconds since view start.
+     */
+    const val FLUTTER_FIRST_BUILD_COMPLETE = "_dd.performance.first_build_complete"
+
+    /**
+     * Custom value for Interaction To Next view.
+     * For Flutter this is the amount of time between an action occurring and the First Build Complete occurring
+     * on the next view.
+     */
+    const val CUSTOM_INV_VALUE = "_dd.view.custom_inv_value"
```

Comment:
type here should be specified explicitly (that is why API surface generation fails)